src/client/Client.h

```diff
@@ -404,6 +408,8 @@ class Client : public Dispatcher, public md_config_obs_t {
 public:
   entity_name_t get_myname() { return messenger->get_myname(); } 
   void _sync_write_commit(Inode *in);
+  void wait_on_list(list<Cond*>& ls);
+  void signal_cond_list(list<Cond*>& ls);
```

Comment:
You could keep this private by declaring Delegation (or Fh, I forget which one is invoking it) as a friend. Not quite sure which is preferable...

Reply:
This seems like a pretty innocuous set of methods to expose.

Reply:
Nah, that's fine, it just stuck out a bit in the diff and I had to think through what was going on.